test/core/scale/scale_variant_test.cpp

```diff
@@ -4,29 +4,31 @@
  */
 
 #include <gtest/gtest.h>
+#include <boost/variant/get.hpp>
+#include <variant>
```

Comment:
Do we still need both variant and boost::variant?

Reply:
no